src/main/resources/templates/fragments/filterbox.html

```diff
@@ -0,0 +1,12 @@
+<div th:fragment="filterBox">
+    <p>Gender</p>
+    <input type="radio" class="radGender" id="male">
+    <label for=male>Male</label><br>
+    <input type="radio" class="radGender" id="female">
+    <label for="female">Female</label><br>
+    <input type="radio" class="radGender" id="other">
+    <label for="other">Other</label><br>
+
+    <button class="Submit" id="SubmitGender">Submit</button>
+    <button class="Cancel" id="CancelGender">Cancel</button>
+</div>
```

Comment:
Is this new functionality?

1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.